src/main/java/com/o19s/es/ltr/feature/store/StoredLtrModel.java

```diff
@@ -79,6 +86,7 @@ public StoredLtrModel(StreamInput input) throws IOException {
         rankingModelType = input.readString();
         rankingModel = input.readString();
         modelAsString = input.readBoolean();
+        this.parsedFtrNorms = new StoredFeatureNormalizers(input);
```

Comment:
this is a breaking change, and will break while doing a rolling restart to ship this version of the plugin.
The last time we had to do this we guarded against the next version this feature will be shipped to, e.g. if this feature is shipped with v7.6.3 writing the normalizers must be guarded with:

        if (input.getVersion().onOrAfter(Version.V_7_6_3)) {
             new StoredFeatureNormalizers(input);
        }

Reply:
I changed this to `after(V_7_6_1)` so I suppose this will be active when we update to 7.6.2